less/index.less

```
@@ -57,3 +57,178 @@ html, body {
     height: 100%;
 }
 
+html {
+	font-size: 62.5%;
+}
+
+//Variables
+@background-color: lightgray;
+@font-family: Tahoma, Helvetica, serif;
+@height: 100%;
+@padding: 10px;
+@font-size: 4rem;
+@width: 45%;
+@flex-direction: column;
+@justify-content: center;
+@table-background-color: rgb(199, 196, 196);
+
+//Mixins
+.middle-left-right {
+	display: flex;
+	flex-direction: column;
+	width: 50%;
+}
+
+// Nested at-rule
+// @media (max-width: 500px) {
+// 		//background: @background-color;
+// 		//.middle-right;
+// 		//.middle-left-right;
+// 		flex-flow: wrap;
+// 		width: 100%;
+
+// 		flex-direction: column;
+// }
+
+.container {
+	font-size: 1.6rem;
+	width: 75%;
+	margin: 0 auto;
+	@media (max-width: 500px) {
+		width: 100%;
+	}
+	
+		header {
+
+			@media (max-width: 500px) {
+				//background: @background-color;
+				//.middle-right;
+				//.middle-left-right;
+				//flex-flow: wrap;
+				width: 100%;
+				//flex-direction: column;
+				display: flex;
+				justify-content: center;
+				max-width: 100%;
+			}		
+
+			nav {
+
+			}
+		}
+	}
+	
+	
+
+
+nav {
+	display: flex;
+	justify-content: space-between;
+	width: 50%;
+	margin-left: 50%;
+	margin-bottom: 20px;
+	
+
+	@media (max-width: 500px) {
+		margin-left: 0%;
+	}		
+	
+	a {
+		text-decoration: none;
+		color: black;
+		justify-content: flex-end;
+	}
+}
+
+img {
+	@media (max-width: 500px) {
+		width: 100%;
+		height: auto;
+	}
+}
+
+.top-content {
+	background: @background-color;
+	padding: @padding;
+
+	h1 {
+		font-size: @font-size;
+		margin-bottom: 10px;
+	}
+}
+
+p {
+	margin: 20px 0;
+}
+
+.middle-section {
+	display: flex;
+	margin-top: 20px;
+
+	@media (max-width: 500px) {
+		flex-direction: column;
+		width: 200%;
+		padding: 2%;
+
+	}
+	
```

```suggestion
.middle-section {
	display: flex;
	margin-top: 20px;

	@media (max-width: 500px) {
		flex-direction: column;
		width: 200%;
		padding: 2%;

	}
```
This is a great place to call your mixin